src/components/Files/FilesTab.tsx

```diff
@@ -586,60 +597,105 @@ const FileUploadDialog = ({
           </DialogTitle>
         </DialogHeader>
         <div className="space-y-4">
-          {fileUpload.files.map((file, index) => (
-            <div key={index} className="space-y-2">
-              <div className="flex items-center justify-between gap-2 rounded-md bg-secondary-300 px-4 py-2">
-                <span className="flex items-center truncate">
-                  <CareIcon icon="l-paperclip" className="mr-2 shrink-0" />
-                  <span className="truncate max-w-xs" title={file.name}>
-                    {file.name}
-                  </span>
-                </span>
-                <Button
-                  variant="ghost"
-                  size="icon"
-                  onClick={() => fileUpload.removeFile(index)}
-                  disabled={fileUpload.uploading}
-                >
-                  <CareIcon icon="l-times" />
-                </Button>
-              </div>
+          {isPdf ? (
+            <>
               <div className="space-y-2">
+                <ul className="list-disc list-inside space-y-1">
+                  {fileUpload.files.map((file, index) => (
+                    <li key={index} className="truncate">
+                      {file.name}
+                    </li>
+                  ))}
+                </ul>
+              </div>
+              <div>
                 <Label
-                  htmlFor={`upload-file-name-${index}`}
-                  className="text-sm font-medium text-gray-700"
+                  htmlFor="upload-file-name-0"
+                  className="block text-sm font-medium text-gray-700"
                 >
-                  {t("enter_file_name")}
+                  {t("enter_combined_file_name")}
```

Comment:
```suggestion
                  {t("enter_file_name")}
```
Let's use the other one as is.